amy/workshops/models.py

```diff
@@ -656,27 +656,21 @@ def passed_either(*reqs):
                 )
             )
 
-        LESSON_CONTRIBUTION_NAMES = [
-            "Lesson Contribution",
-            "SWC Homework",
-            "DC Homework",
-            "LC Homework",
-        ]
-        DEMO_TRAININGPROGRESS_NAMES = ["Demo", "SWC Demo", "DC Demo", "LC Demo"]
+        LESSON_CONTRIBUTION_NAMES = ["Lesson Contribution"]
 
         return self.annotate(
             passed_training=passed("Training"),
             passed_lesson_contribution=passed_either(*LESSON_CONTRIBUTION_NAMES),
```

Comment:
Can we use `passed` instead of `passed_either` with literal string `"Lesson Contribution"` as an argument?

Reply:
Yes - this is coming in a later PR which updates the checkout requirements after the change from "Lesson Contribution" -> "Get Involved"